notes.md

```diff
@@ -31,9 +31,9 @@ professional networking and grant proposal writing.
 - [Building Binaryen without `cmake`](#user-content-building-binaryen-without-cmake)
 - [Dividing the FFI between crates](#user-content-dividing-the-ffi-between-crates)
 - [Linking to crates that contain no Rust code](#user-content-linking-to-crates-that-contain-no-rust-code)
-- [Calling the C++ `main` function from Rust](#user-content-calling-c++-main-function-from-rust)
+- [Calling the C++ `main` function from Rust](#user-content-calling-the-c-main-function-from-rust)
 - [`cxx` and Binaryen](#user-content-cxx-and-binaryen)
-- [Our C++ shim layer](#user-content-our-c++-shim-layer)
+- [Our C++ shim layer](#user-content-our-c-shim-layer)
```

Comment:
I don't know if you plan to publish this writeup to a blog somewhere other than GitHub; you'll want to check what it does with this section heading. The one in the PR works for GitHub; on master it's currently a broken link: https://github.com/brson/wasm-opt-rs/blob/aad563327f248190a17ad48de8e0a3f667b76597/notes.md#table-of-contents.

Reply:
It'll be on my blog, and yeah I'll have to adjust these header links.